index.js

```diff
@@ -1,8 +1,10 @@
-import translate from './core/translate.js';
-import defaultTheme from './core/theme.js';
+import translate from './translate.js';
+import defaultTheme from './theme.js';
 import merge from './util/merge.js';
 import { css } from './util/otion.js';
 
+export { setup, hydrate } from './util/otion.js';
```

Comment:
Do you want to expose a separate entry point for SSR so you won't bloat your runtime unnecessarily?

Reply:
Oh, wait. Nevermind. That's coming from the otion runtime anyway. :)